src/Analyzers.CodeFixes/CSharp/CodeFixes/UseExplicitlyOrImplicitlyTypedArrayCodeFixProvider.cs

```
@@ -39,29 +55,40 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         Document document = context.Document;
         Diagnostic diagnostic = context.Diagnostics[0];
 
-        switch (node)
+        string title = node switch
+        {
+            ImplicitArrayCreationExpressionSyntax => "Use explicitly typed array",
+            ArrayCreationExpressionSyntax => "Use implicitly typed array",
+            _ => "",
```

This arm is unreachable, so I would prefer to throw InvalidOperationException.